pp/integrations/plugin-packs/procedures/operatingsystems-windows-telegraf-agent.md

```diff
@@ -97,35 +99,42 @@ No metrics for this service.
 
 ## Prerequisites
 
-### Centreon Engine
+### Network flow
+
+Two TCP flows must be open from the host to the poller.
+
+| Source | Destination    | Protocole | Port | Purpose |
+| ------ | -------------- | --------- | ---- | --- |
+| Hôte supervisé | Collecteur  | TCP       | 1443 | Access to the configuration of the Telegraf agent. |
+| Hôte supervisé | Collecteur  | TCP       | 4317 | OpenTelemetry data flow. |
+
+### System prerequisites on the poller
 
-> To be able to use the Telegraf agent, you must use a poller with at least version `24.04.2` of `centreon-engine`.
+> To be able to use the Telegraf agent, you must use a poller with at least version `24.04.2` of `centreon-engine`. The Telegraf agent will configure itself via a HTTPS request sent to Centreon Engine.
 
-The Telegraf agent will configure itself via a HTTPS request sent to Centreon Engine.
+1. For this to work, you must first get a valid certificate or generate a self-signed one on the poller as detailed below.
 
-1. For this to work, you must first generate a certificate on the poller:
+> In the command below, replace `${HOSTNAME}` with the poller's FQDN if they don't match.
 
 ```bash
-openssl req -new -newkey rsa:2048 -sha256 -days 3650 -nodes -x509 -keyout /etc/centreon-engine/conf-server.key -out /etc/centreon-engine/conf-server.crt
+openssl req -new -subj "/CN=${HOSTNAME}" -addext "subjectAltName = DNS:${HOSTNAME}" -newkey rsa:2048 -sha256 -days 365 -nodes -x509 -keyout /etc/centreon-engine/conf-server.key -out /etc/centreon-engine/conf-server.crt
 chown centreon-engine: /etc/centreon-engine/conf-*
 ```
 
-> **Important** : give the IP address or the FQDN of the poller when the **Common Name** information is prompted.
->
-> Example:
-> ```
-> Common Name (e.g. server FQDN or YOUR name) []:172.16.5.166
-> ```
+> The `-days 365` option limits the certificate's validity to one year. You may choose a longer or shorter duration according to your security/maintainance preferences.
 
-2. Then prepare the information Engine will have to provide to the Telegraf agent.
+2. Then define Engine's configuration server for Telegraf.
```

Comment:
```suggestion
2. Then provide Engine with the connection information it needs to give to the Telegraf agent so that it can send information to Engine.
```